app/lib/actions/katello/repository/sync.rb

```diff
@@ -25,11 +25,11 @@ def plan(repo, options = {})
           validate_contents = options.fetch(:validate_contents, false)
           skip_metadata_check = options.fetch(:skip_metadata_check, false) || (validate_contents && repo.yum?)
           generate_applicability =  options.fetch(:generate_applicability, repo.yum? || repo.deb?)
-
+          skip_candlepin_check = options.fetch(:skip_candlepin_check, false)
```

Comment:
This param is meant to be internal for us or support to debug right? If not should we expose that param on the controller side so hammer picks it up?